### ISSUE TITLE
CodePush SourceMap-Output doesn't work with Output-Dir

### ISSUE THREAD

**Issue by TheCodeKing** [Microsoft/appcenter-cli#504]
I'm trying to generate a CodePush bundle for a React-Native project, and write out the source maps to a folder outside of the CodePush location. I'm using version 1.1.7.

When I specify the source maps location, it's completely ignored and maps get bundled into the CodePush package. Is there ever a scenario where you would distribute source maps to users?

```
appcenter codepush release-react -a "test/Test-App" 
  --deployment-name Staging 
  --entry-file index.js 
  --output-dir "/tmp/output" 
  --sourcemap-output "/tmp/maps/main.jsbundle.map"
```

If I remove the `--output-dir` param then it works, but then I can't set the bundle output location which I also need.

```
appcenter codepush release-react -a "test/Test-App" 
  --deployment-name Staging 
  --entry-file index.js 
  --sourcemap-output "/tmp/maps/main.jsbundle.map"
```

If I try and specify the `--output-dir` without `--sourcemap-output`, then the source maps still get generated.

```
appcenter codepush release-react -a "test/Test-App" 
  --deployment-name Staging 
  --entry-file index.js 
  --output-dir "/tmp/output" 
```

**Comment by TheCodeKing** [comment]
Problem seems to be this line

https://github.com/Microsoft/appcenter-cli/blob/7759122a660276834f2e8a6203893c0942f8623b/src/commands/codepush/release-react.ts#L131

Should be:

```
if (this.outputDir && !this. sourcemapOutput) {
```

**Comment by amchew** [comment]
Thanks @TheCodeKing for flagging this to us! We'll investigate and get back shortly.

**Comment by alexandergoncharov** [comment]
Hi @TheCodeKing, @amchew ,

@TheCodeKing , Good catch! I removed this `if` block to prevent forced generating sourcemaps and prepared PR: https://github.com/Microsoft/appcenter-cli/pull/516
Could you please review it and let me know if you have any questions or suggestions.

Thanks,
Alexander

**Comment by amchew** [comment]
Hi @TheCodeKing - could you please review and let us know if you have any questions? If not, let's proceed to merge it on Monday end of day 12/17 @alexandergoncharov.

**Comment by TheCodeKing** [comment]
@alexandergoncharov looks all good from my side. Thanks for the fix :)

**Comment by amchew** [comment]
Hi @TheCodeKing! We released a fix to this issue in CLI [v1.1.10](https://github.com/Microsoft/appcenter-cli/releases/tag/v1.1.10). Please go ahead and download the new version. I'll be closing this ticket for now. Let us know if you have any other questions!
